Google.PowerShell.IntegrationTests/SQL/testsql.csv

```diff
@@ -0,0 +1,2 @@
+"first guest","I got here!",1
```

Comment:
"testsql.csv" is not a descriptive name for a file. How about "sample-table.csv" or something?